function.dd

```diff
@@ -123,7 +123,7 @@ $(H4 $(LNAME2 pure-functions, Pure Functions))
 
         $(P Pure functions are functions which cannot access global or static, mutable
         state save through their arguments. This can enable optimizations based on the fact
-        that a pure function is guaranteed to mutate nothing which isn't passed to it,
+        that a pure function is guaranteed to mutate nothing which is not passed to it,
```

Comment:
The (previous) phrasing is quite awkward. Let's take this opportunity to change the text to:

```
This enables optimizations based on the fact that a pure function may at most mutate state reachable through its parameters.
```